dev/Vagrantfile

```diff
@@ -110,6 +127,17 @@ def define_chef_server(config, attributes)
       json = simple_deep_merge(json, { "provisioning" => { "opscode-reporting-config" => pg } })
     end
 
+    if attributes['vm']['ldap']['start'] and attributes['vm']['ldap']['use-external']
```

Comment:
In this case, the additional external flag isn't buying us anything - I kind of see this as more we're using LDAP and need the ldap server, or we aren't/don't.  

'use-external' originally came about indicate that server should be configured to use external versions of things that ship with it (postgres).